bazel/markdown_to_html/src/tranformations/list.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import {RendererApi} from 'marked';
+
+export const list: RendererApi['list'] = (body, ordered, start) => {
+  if (ordered) {
+    return `
+    <ol class="docs-ordered-list">
+      ${body}
+    </ol>
+    `;
```

Comment:
```suggestion
    return `<ol class="docs-ordered-list">${body}</ol>`;
```
optional: seems like these would fit on one line

Reply:
They do fit, but since they are used to determine what is rendered, its on multiple lines to set up the formatting of the output.  I just kept as is from the original, but am happy to reconsider how we want to do it.  Just wanted to let you know the thinking of it.